CfdOF/CfdTools.py

```diff
@@ -723,11 +725,24 @@ def getRunEnvironment():
         return {}
 
 
-def makeRunCommand(cmd, dir, source_env=True):
+def makeRunCommand(cmd, dir, source_env=True, usedocker=False):
```

Comment:
To avoid passing in the usedocker parameter in so many places, could we maybe make this a separate runtime type (similar to the existing 'WindowsDocker' type)?

Reply:
Resolved in #66